source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```diff
@@ -55,103 +96,127 @@ content: |
         :tabid: java-sync
 
         .. code-block:: java
-           :emphasize-lines: 7-9, 11
 
-           BsonString masterKeyRegion = new BsonString("<Master Key AWS Region>"); // e.g. "us-east-2"
-           BsonString awsAccessKeyId = new BsonString("<IAM User Access Key ID>");
-           BsonString awsSecretAccessKey = new BsonString("<IAM User Secret Access Key>");
+           // TODO: check correctness
            Map<String, Map<String, Object>> kmsProviders = new HashMap<String, Map<String, Object>>();
            Map<String, Object> providerDetails = new HashMap<String, Object>();
 
-           providerDetails.put("accessKeyId", awsAccessKeyId);
-           providerDetails.put("secretAccessKey", awsSecretAccessKey);
-           providerDetails.put("region", masterKeyRegion);
+           providerDetails.put("email", new BsonString("<GCP service account email>"));
+           providerDetails.put("privateKey", new BsonString("<GCP service account private key>"));
+           providerDetails.put("endpoint", new BsonString("<GCP authentication endpoint>"));
+
+           kmsProviders.put("gcp", providerDetails);
 
-           kmsProviders.put("aws", providerDetails);
      .. tab::
         :tabid: nodejs
 
         .. code-block:: javascript
 
+           // TODO: check correctness
            kmsProviders = {
-             aws: {
-               accessKeyId: '<IAM User Access Key ID>',
-               secretAccessKey: '<IAM User Secret Access Key>',
+             gcp: {
+               email: '<GCP service account email>',
+               privateKey: '<GCP service account private key>',
+               endpoint: '<GCP authentication endpoint>',
              }
            }
+
      .. tab::
         :tabid: python
 
         .. code-block:: python
 
-           kms_providers = {
-               "aws": {
-                   "accessKeyId": "<IAM User Access Key ID>",
-                   "secretAccessKey": "<IAM User Secret Access Key>"
+           kms_provider = {
+               "gcp": {
+                   "email": "<GCP service account email>",
+                   "privateKey": "<GCP service account private key>",
+                   "endpoint": "<GCP authentication endpoint>",
                }
            }
 ---
 title: Create a New Data Encryption Key
-ref: create-a-new-data-key
+ref: create-a-new-data-key-gcp
 content: |
-  The following diagram shows how the **customer master key** is created and
-  stored when using a KMS provider:
+  In this step, we generate a new **data encryption key** using the
```

Comment:
Suggestion: Try to avoid "we", as it places the emphasis on the writer and not the reader. [ref](https://docs.mongodb.com/meta/style-guide/writing/write-to-the-user/)

Perhaps: "This step generates a new data encryption key..."

Reply:
Will address. Would definitely like to discuss implications of "we" at some point. @atsansone